### PR TITLE
Update 01-Introduction.Rmd

### PR DESCRIPTION
**Pull request by j-sayers**
Just want to make sure I can continue to add commits / coments to this pull request while open

### CODE REVIEW

File: 01-Introduction.Rmd [hunk 1]
```diff
@@ -23,7 +23,7 @@ What can you do on the NatureCounts [webportal](http://www.naturecounts.ca)?
 
 ## NatureCounts and R {#intro1.1}
 
-The naturecounts R package was create to allow users to directly access and download data from NatureCounts into R. The goal of this online book is to show NatureCounts users how to use the [R](https://www.r-project.org/) statistical programming language  to access, filter, summarize, and visualize naturecounts data. Moving forward, we intend to create more advanced tutorials which explore analytical procedures suitable for various datasets. If you have suggestions for additional content, please let us know by emailing dethier@birdscanada.org
+The naturecounts R package was create to allow users to directly access and download data from NatureCounts into R. The goal of this online book is to show NatureCounts users how to use the [R](https://www.r-project.org/) statistical programming language  to access, filter, summarize, and visualize NatureCounts data. Moving forward, we intend to create more advanced tutorials which explore analytical procedures suitable for various datasets. If you have suggestions for additional content, please let us know by emailing dethier@birdscanada.org
```
Comment (j-sayers) [inline 01-Introduction.Rmd]: Just capitalized NatureCounts to make it consistent with others

Comment (DMEthier) [reply 01-Introduction.Rmd]: OK

**Review by DMEthier — APPROVED** [review]
Changes

**Comment by DMEthier** [comment]
No comment

**Pull request merged**